contracts/mocks/docs/ERC4626Fees.sol

```diff
@@ -7,36 +7,41 @@ import {ERC4626} from "../../token/ERC20/extensions/ERC4626.sol";
 import {SafeERC20} from "../../token/ERC20/utils/SafeERC20.sol";
 import {Math} from "../../utils/math/Math.sol";
 
+/// @dev ERC4626 vault with entry/exit fees expressed in https://en.wikipedia.org/wiki/Per_cent_mille[PCM].
```

Comment:
I expect bps to be more common in finance than PCM. Are we sure we don't want to fix the scale constant to be actual bps?

Reply:
We can go to BPS. The actual cahnges to the code will be smaller, but the have to change the unit tests instead.

Reply:
Good data points shared here by @Amxx showing that bps might be more used: https://forum.openzeppelin.com/t/please-explain-erc4626fees-math/37141/5

Reply:
Thanks for checking it out, I completely missed taking a look.
I reverted back to base point instead.